### PR TITLE
Content, Formatting, and Image Size Edits on all Data Visualization Patterns

### PR DESCRIPTION
**Pull request by jennyhaines**
@LHinson 
## Description
I have made edits to all data visualizations, as specified in this [Jira Story](https://patternfly.atlassian.net/browse/PTNFLY-2117).

## Changes

Write a list of changes the PR introduces

[Bullet Chart](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/bullet-chart/design)
* Remove the title “Overview”
* We need anchor links like the other pages
* Images are large on the design tab 
* Remove title at the top of the page on the design tab
* Remove title “Examples”
* Optional is bold on the design tab
* Don’t need that many bullets 
* images names should all be lower case
* image name cleanup 

[Donut Chart](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/donut-chart/design)
* Remove the title “overview”
* Images are too big 
* The images on the design tab don’t work with this 1 column format
* On Overview, Remove title “Examples”
* Bullet format error on design tab
* clean up image names 

[Heat Map](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/heat-map/design)
* Images sizes need fixed
* Remove title “Examples”
* Line Chart
* Remove “Overview” title 
* Images sizes too big
* Remove “Description”
* Fix images on design tab - doesn’t work with this 1 column layout 
* Remove title “Examples”
* Bullet format error on design tab
* clean up image names (similar approach as above) 

[Pie Chart](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/pie-chart/design)
* Remove “Overview” 
* Images are too big 
* Image layout does not work with this 1 column format 
* Remove title “Examples”’
* Bullet format error on design tab
* clean up image names (similar approach as above) 

[Timeline](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/timeline)
* Remove “Overview” 
* Remove “Examples”
* Remove “Timeline” on design tab
* Make Interaction bullets one line on itself
* clean up image names 

[Utilization Bar Chart](https://github.com/patternfly/patternfly-design/tree/master/pattern-library/data-visualization/utilization-bar-chart/design)
* Remove “Overview”
* The images sizes are too large 
* Bullet format bug on design tab
* clean up image names

### CODE REVIEW

File: pattern-library/data-visualization/donut-chart/design/overview.md [hunk 1]
```diff
@@ -1,25 +1,22 @@
 # Donut Chart
 
-## Overview
 The most common use case for a donut chart are:
-
-Show progress completion as a percentage.
-Show utilization for an object (e.g. network, CPU, storage, etc.) as a percentage.
-Show the relationship of a set of values to a whole (design and specs for this use case are not addressed at this time, TBD.).
```
Comment (LHinson) [inline pattern-library/data-visualization/donut-chart/design/overview.md]: whoop, just noticed a bug... you can remove the text in parenthesis as this is defined/speced in the documentation.

File: pattern-library/data-visualization/pie-chart/design/design.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
   1. **Pie Chart Fill:**
     - For recommendations on fill colors, see the [Color Palette](https://www.patternfly.org/styles/color-palette/).
-    - **Interaction** (optional)**:**  
+    - **Interaction** (optional):  
       -  If drill down behavior is supported, clicking on a segment of the pie will navigate to the appropriate page.
       - If supported, right clicking on a segment of the pie chart will bring up a menu with associated actions.
 
-  1. **Label** (optional)**:**
+  1. **Label** (optional):
```
Comment (LHinson) [inline pattern-library/data-visualization/pie-chart/design/design.md]: does not need to be a bullet

File: pattern-library/data-visualization/pie-chart/design/design.md [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
     - The label may be shown either on top or bottom of the pie chart.
 
-  1. **Tooltip** (optional)**:**
+  1. **Tooltip** (optional):
     - We recommend that the name and percentage are displayed on hover.
```
Comment (LHinson) [inline pattern-library/data-visualization/pie-chart/design/design.md]: does not need to be a bullet

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md [hunk 4]
```diff
@@ -10,14 +9,11 @@
...
-  1. **Background Fill:**
-    - The background color is grey (#d1d1d1).
+  1. **Background Fill:** The background color is grey (#d1d1d1).
 
-  1. **Tooltip:**
-    - It is recommend that the percentage is displayed on hover.
+  1. **Tooltip:** It is recommend that the percentage is displayed on hover.
 
-  1. **Warning Threshold Indicator** (optional)**:**
+  1. **Warning Threshold Indicator** (optional):
```
Comment (LHinson) [inline pattern-library/data-visualization/utilization-bar-chart/design/design.md]: no need to be a bullet

**Comment by LHinson** [comment]
Thanks @jennyhaines Some comments above.

**Comment by LHinson** [comment]
bitmoji

gonna merge!

**Comment by jennyhaines** [comment]
Thanks Leslie!

*Jenny Haines*
*UI/VISUAL DESIGNER*
(m) 443-889-2881
jhaines@redhat.com
jennyhaines10@gmail.com

**Pull request merged**